frontend/src/lib/components/entityDetails/InfoItem.svelte

```diff
@@ -24,10 +25,20 @@ Used to show a label-content pair in a Candidate's basic information.
 ```
 -->
 
-<div class="infoItem grid grid-flow-col grid-cols-[min-content_auto] justify-start gap-md">
+<div class="grid justify-start gap-md {vertical ? 'vertical-grid' : 'horizontal-grid'}">
```

Comment:
I also tried using [conditional CSS directives from Svelte](https://sveltebyexample.com/class-and-style-directives/) but eventually settled on just a regular conditional (ternary) operator combined with a string insertion.

My reasoning:
- it allow to apply different classes at once (either/or) rather than have to conditional CSS Svelte directives like so:

```
class:vertical-grid={vertical}
class:horizontal-grid={!vertical}
```
- conditional CSS directives are not possible to define when one wants to apply multiple classes based on a single condition, whereas the approach with ternary operator is not limited in this case, so it is more flexible and therefor better for uniformity